crates/core/src/instance.rs

```diff
@@ -158,6 +159,7 @@ impl Instance {
         sub_context.set_state(self.state.clone());
         sub_context.action_channel = self.action_channel.clone();
         sub_context.observer_channel = self.observer_channel.clone();
+        FuturesDiagnosticTrace::run(sub_context.future_trace.clone());
```

Comment:
Can we make this conditional so we can decide through the conductor config if we want to run this?